services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/ApplicationConfig.java

```diff
@@ -277,10 +308,21 @@ public AmqpEndpoint credentialsAmqpEndpoint() {
         // need to use backend service because the Credentials service's "get" operation
         // supports auto-provisioning and thus needs to be able to create a device on the fly
         final MongoDbBasedDeviceBackend service = new MongoDbBasedDeviceBackend(registrationService(),
-                credentialsService());
+                                                                                credentialsService());
```

Comment:
Could you please revert it back to the original version? The previous formatting was fine.

Reply:
reverted